roles/variables/defaults/main.yml

```diff
@@ -1770,16 +1853,16 @@ mds_health_check_user: "{{mds_super_user}}"
 mds_health_check_password: "{{mds_super_user_password}}"
 
 ### User for authenticated Kafka Admin API Health Check.
-kafka_broker_rest_health_check_user: "{{ mds_super_user if rbac_enabled|bool else kafka_broker_rest_proxy_basic_users.admin.principal }}"
+kafka_broker_rest_health_check_user: "{{ kafka_broker_rest_proxy_basic_users.admin.principal }}"
```

Comment:
this variable was used in basic authentication earlier. now that authentication is done using bearer token. So now is this variable used anywhere

Reply:
which variable are you referring to?

Reply:
kafka_broker_rest_health_check_user

Reply:
this is used for authentication in case of non rbac cluster as well, we still have that task in health check. In case of rbac cluster, this variable is no longer needed

Reply:
okay makes sense